src/zenml/integrations/label_studio/label_config_generators/label_config_generators.py

```diff
@@ -0,0 +1,89 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Implementation of label config generators for Label Studio."""
+
+from typing import List, Tuple
+
+from zenml.enums import AnnotationTasks
+from zenml.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+TASK_TO_FILENAME_REFERENCE_MAPPING = {
+    str(AnnotationTasks.IMAGE_CLASSIFICATION): "image",
```

Comment:
Any reason this needs to be a `str` key and can't just be the enum?

Reply:
`mypy` was complaining.

Reply:
Mhm that's strange, I don't see any reason why it would complain about enum as dict keys.
I'd still suggest using `AnnotationTasks.IMAGE_CLASSIFICATION.value` here instead of converting that to a string using `str(...)`.